src/components/Employees/index.tsx

```diff
@@ -0,0 +1,214 @@
+import styles from './Employees.module.css';
+import employees from '@components/Employees/employeesData';
+import EmployeeItem from '@components/EmployeeItem';
+import { IEmployeeItem } from '@components/EmployeeItem/types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearch, faTimesCircle } from '@fortawesome/free-solid-svg-icons';
+import { useSearchParams } from 'react-router-dom';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+function Employees() {
+    const { t } = useTranslation();
+    const [searchParams, setSearchParams] = useSearchParams({});
+    const [searchActiveButtons, setSearchActiveButtons] = useState<string[]>(
```

Comment:
Please rename to selectedFilters

Reply:
I will change. It started like buttons but later on they got better sense. I will rename all functions and parameters to get more sense.